docs/release-notes/14-4-0/README.md

```diff
@@ -0,0 +1,224 @@
+---
+title: OpenProject 14.4.0
+sidebar_navigation:
+    title: 14.4.0
+release_version: 14.4.0
+release_date: 2024-08-14
+---
+
+# OpenProject 14.4.0
+
+Release date: 2024-08-14
+
+We released [OpenProject 14.4.0](https://community.openproject.org/versions/2063). The release contains several bug fixes and we recommend updating to the newest version. 
+
+In these Release Notes, we will give an overview of important technical updates as well as important feature changes. At the end, you will find a complete list of all changes and bug fixes.
+
+## Important technical updates
+
+### Extend API authentication to accept JWT issued by OpenID provider to other client
+
+OpenProject 14.4 introduces a new feature that allows OpenID clients, such as Nextcloud servers, to use access tokens obtained from an OpenID provider, like Keycloak, as an authentication mechanism for the OpenProject API. This enhancement **enables users to skip the OAuth grant flow**, streamlining the authentication process.
+
+With this feature, the OpenProject API will validate access tokens issued by the OpenID provider (Keycloak) by checking the token's signature and authenticating the user using the sub claim value. This integration ensures secure and efficient API authentication for OpenID clients.
+
+For more details, see https://community.openproject.org/wp/55643.
```

Comment:
Maybe also link to the documentation here: /docs/api/introduction/#oidc-provider-generated-jwt-as-a-bearer-token